local/README.md

```diff
@@ -0,0 +1,25 @@
+# Local Execution
```

Comment:
Can we add this under the Testing section in the README.md instead of introducing a new dir structure? We could include a note that it’s specific to VS Code, or alternatively, provide a more generic example like the one below:
```bash
PLUGIN_GLOB="**/*.yml" \
PLUGIN_EXCLUDES="**/b.*" \
PLUGIN_DIR="$(pwd)" \
DRONE_OUTPUT="$(pwd)/drone_output.properties" \
go run main.go
```
Also remember, this approach works when the code is written using only native Go libraries. in cases where external commands (e.g., ssh) are executed using os.Exec, it’s recommended to test inside a container image.

Reply:
Done